README.md

```diff
@@ -203,41 +203,41 @@ We use Docker in this project to assemble a different environment from different
+:warning: When the console is ready again and the command line appears, you will have to wait __another 1-2 minutes__ until REDAXO is fully installed. You will not see the status of the REDAXO installation in your console because the process is taking place in the container. You can look at the container logs using `docker-compose logs web` (The `web` at the end is our web server, `db` would be the database). Alternatively you can see the logs in the free Docker tool [Kitematic](https://kitematic.com), This is very useful when working with multiple Docker projects.
 
-Danach steht dir ein frisches REDAXO inkl. [Demo-Website](https://github.com/FriendsOfREDAXO/demo_base) im Browser zur Verfügung unter:
+Then you have a fresh REDAXO whit the. [Demo-Website](https://github.com/FriendsOfREDAXO/demo_base) available in the browser under:
 
     http://localhost:20080
 
-Ins REDAXO-Backend kannst du dich einloggen mit `admin`/`admin`.
+You can log in to the REDAXO backend with `admin`/`admin`.
 
 :tada:
 
-### Wie geht es weiter?
+### What are the next steps?
 
-Du solltest dich etwas mit Docker beschäftigen und vertiefst dich am besten in die [offizielle Dokumentation](https://docs.docker.com). Lass dich dabei nicht abschrecken, denn Docker kann furchtbar kompliziert werden, wenn man es in großem Stil nutzt. Und selbst in unserem kleinen Kontext ist nicht alles ganz einfach zu verstehen. Mit diesem Setup hast du eine funktionierende Entwicklungsumgebung für REDAXO, die du nach und nach im Detail verstehen wirst, wenn du dich länger mit Docker beschäftigst!
+You should work a bit whit Docker, and take a look at the [official Documentation](https://docs.docker.com). Do not let that scare you off, because Docker can get really complicated if you use it on a big scale. And even in our small context, not everything is easy to understand. With this setup, you'll have a working development environment for REDAXO that you'll understand in more detail as you work longer with Docker!
 
-### Welche Funktion haben die Dateien und Ordner dieses Pakets?
+### What is the function of the files and folders of this package?
 
-Wir gehen mal von oben nach unten durch:
+Let's take look step by step:
 
-#### Datenbank
+#### Database
 
     db/
     
-In diesen Ordner wird die __Datenbank__ des Containers _persistiert_, also dauerhaft auf deinem System gespeichert. Würden wir das nicht machen, wäre die Datenbank jedesmal aufs Neue leer, wenn du den Container baust. Weil wir aber dauerhaft am REDAXO arbeiten wollen, das sich in diesem Paket befindet, müssen wir die Datenbank außerhalb des Containers hinterlegen.
+In this folder, the __Database__ of the container _persists_, ie permanently stored on your system. If we did not do that, the database would be empty again each time you build the container. But because we want to work permanently on the REDAXO, which is in this package, we have to deposit the database outside the container.
```

Comment:
In this folder, the __Database__ of the container _persists_, ie is permanently stored on your system. If we did not do that, the database would be empty again each time you build the container. But because we want to work permanently on the REDAXO inside this package, we have to keep the database outside the container.